charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,64 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/ash
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/dav_event_handler"
+    while IFS= read -r line; do
+      status=""
+      type=""
+      path=""
+      dest=""
+
+      for i in $line
+      do
+        if [ "${status}" == "" ]
+        then
+          status="${i}"
+        elif [ "${type}" == "" ]
+        then
+          type="${i}"
+        elif [ "${path}" == "" ]
+        then
+          path="${i}"
+        elif [ "${dest}" == "" ]
+        then
+          dest=$(echo "${i}" | awk '{i = index($1,"/webdav/"); out = substr($1, i); print out}')
+        fi
+      done
+
+      matched=$(expr match "${path}" '/webdav.*')
+      if [ \( "${type}" == "PUT" -o "${type}" == "DELETE" -o "${type}" == "MKCOL" -o "${type}" == "MOVE" -o "${type}" == "COPY" \) -a ${matched} != 0 ]
+      then
+        OP="Unknown"
+        if [ "${status}" == "201" -a "${type}" == "PUT" ]
+        then
+          OP="NEW"
+        elif [ "${status}" == "201" -a "${type}" == "MKCOL" ]
+        then
+          OP="MKDIR"
+        elif [ "${status}" == "201" -a "${type}" == "MOVE" ]
+        then
+          OP="MOVE"
+        elif [ "${status}" == "201" -a "${type}" == "COPY" ]
+        then
+          OP="COPY"
+        elif [ "${status}" == "204" -a "${type}" == "PUT" ]
+        then
+          OP="EDIT"
+        elif [ "${status}" == "204" -a "${type}" == "DELETE" ]
+        then
+          OP="DELETE"
+        fi
+
+        if [ "${dest}" == "-" ]
+        then
+          echo "Send notification to $ENDPOINT: $OP ${path}" > /proc/self/fd/1
+        else
+          echo "Send notification to $ENDPOINT: $OP ${path} -> ${dest}" > /proc/self/fd/1
+        fi
+        #RES_CODE=$(curl -sw "%{http_code}\n" --connect-timeout 1 -o /dev/null ${ENDPOINT})
```

Comment:
Results of the internal meeting to define requirements:

Payload (send with cURL): JSON Object
{
"mount":"mount",
"operation":"CREATE, DELETE, UPDATE, MOVE",
"path":"origin relative path",
"destination":"optional for move"
}